unittest/IlmCtl/CMakeLists.txt

```diff
@@ -57,7 +56,14 @@ file(
         testVarying.ctl
         testVaryingLookup.ctl
         testVaryingReturn.ctl
-        testVSArrays.ctl
-    DESTINATION
-        ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
-)
+        testVSArrays.ctl)
+        
+if (MSVC)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Release)
```

Comment:
Do we need to copy files in both release and debug directories everytime? Instead would it be better to copy in respective directories based on the target cmake build type?